Cesium.Compiler/stdlib/stdarg.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+typedef void* va_list;
+
+#define va_start(list,...) list = __varargs
+#define va_copy(dest, src) dest = src;
```

Comment:
I've decided to remove `;` from here, to force the user to put `;` on their own.